client/src/components/Logout/__tests__/index.spec.js

```diff
@@ -1,7 +1,6 @@
 import React from 'react';
 import { create } from 'react-test-renderer';
 import { mount } from 'enzyme';
-import moxios from 'moxios';
```

Comment:
Maybe this dependency should be deleted from package.json if we do not use it anymore